lib/SILOptimizer/Mandatory/AddressLowering.cpp

```diff
@@ -2080,6 +2094,34 @@ void ApplyRewriter::rewriteApply(ArrayRef<SILValue> newCallArgs) {
   // will be deleted with its destructure_tuple.
 }
 
+void ApplyRewriter::convertBeginApplyWithGuaranteedYield() {
+  auto *origCall = cast<BeginApplyInst>(apply.getInstruction());
+  SmallVector<SILValue, 4> opValues;
+  auto argOps = origCall->getArgumentOperands();
+  llvm::transform(argOps, std::back_inserter(opValues),
+                  [](Operand &op) { return op.get(); });
```

Comment:
I know this is done elsewhere, so it's your choice. But it's a lot easier to read and debug, and faster to build range-based for loops:
`for (auto &oper : getArgumentOperand()) { opValues.push_back(oper.get()); }`

Reply:
I prefer this version as well. But I think I started using the llvm::transform version after getting feedback a couple of times.